From 1d2a3ffda89d2625e40416b2737d941934859057 Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Sun, 23 Mar 2014 20:29:15 -0700
Subject: ASoC: rcar: subnode tidyup for renesas,rsnd.txt

rcar_sound,ssi/src/dai subnode documentation
become more cleaner

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Signed-off-by: Mark Brown <broonie@linaro.org>
(cherry picked from commit 8bab0dd58037623b723b768ee2eb1f7dd1ad0416)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 Documentation/devicetree/bindings/sound/renesas,rsnd.txt | 15 ++++++++++++---
 1 file changed, 12 insertions(+), 3 deletions(-)

diff --git a/Documentation/devicetree/bindings/sound/renesas,rsnd.txt b/Documentation/devicetree/bindings/sound/renesas,rsnd.txt
index 7c6d33f29796..a44e9179faf5 100644
--- a/Documentation/devicetree/bindings/sound/renesas,rsnd.txt
+++ b/Documentation/devicetree/bindings/sound/renesas,rsnd.txt
@@ -7,14 +7,23 @@ Required properties:
 				  required register is
 				   SRU/ADG/SSI      if generation1
 				   SRU/ADG/SSIU/SSI if generation2
-- rcar_sound,ssi		: SSI subnode
-- rcar_sound,scu		: SCU subnode
-- rcar_sound,dai		: DAI subnode
+- rcar_sound,ssi		: Should contain SSI feature.
+				  The number of SSI subnode should be same as HW.
+				  see below for detail.
+- rcar_sound,src		: Should contain SRC feature.
+				  The number of SRC subnode should be same as HW.
+				  see below for detail.
+- rcar_sound,dai		: DAI contents.
+				  The number of DAI subnode should be same as HW.
+				  see below for detail.
 
 SSI subnode properties:
 - interrupts			: Should contain SSI interrupt for PIO transfer
 - shared-pin			: if shared clock pin
 
+SRC subnode properties:
+no properties at this point
+
 DAI subnode properties:
 - playback			: list of playback modules
 - capture			: list of capture  modules
-- 
2.1.2

